scripts/prep/aqm/exevs_aqm_prep.sh

```diff
@@ -63,17 +63,32 @@ while [ ${ic} -le ${endvhr} ]; do
     vldhr=$(printf %2.2d ${ic})
     checkfile=${DCOMINairnow}/${INITDATE}/airnow/${HOURLY_INPUT_TYPE}_${INITDATE}${vldhr}.dat
     if [ -s ${checkfile} ]; then
-        export VHOUR=${vldhr}
-	if [ -s ${conf_dir}/Ascii2Nc_hourly_obsAIRNOW.conf ]; then
-            run_metplus.py ${conf_dir}/Ascii2Nc_hourly_obsAIRNOW.conf ${PARMevs}/metplus_config/machine.conf
-	    export err=$?; err_chk
-	    if [ ${SENDCOM} = "YES" ]; then
-                cpfile=${PREP_SAVE_DIR}/airnow_hourly_aqobs_${INITDATE}${VHOUR}.nc 
-                if [ -s ${cpfile} ]; then cp -v ${cpfile} ${COMOUTproc}; fi
-	    fi
+        screen_file=${DATA}/checked_${HOURLY_INPUT_TYPE}_${INITDATE}${vldhr}.dat
+        python ${USHevs}/${COMPONENT}/screen_airnow_obs_hourly.py ${checkfile} ${screen_file}
+        number_of_record=$(wc -l ${screen_file} | awk -F" " '{print $1}')
+        ## There is 1 header lines 
+        if [ ${number_of_record} -gt 1 ]; then
+            export VHOUR=${vldhr}
+            if [ -s ${conf_dir}/Ascii2Nc_hourly_obsAIRNOW.conf ]; then
+                run_metplus.py ${conf_dir}/Ascii2Nc_hourly_obsAIRNOW.conf ${PARMevs}/metplus_config/machine.conf
```

Comment:
I believe there should be a check here for `$screen_file` before running METplus.